lib/ts/utils.ts

```diff
@@ -210,3 +210,13 @@ export function validateAndNormaliseInputOrThrowError(options: InputType): Norma
         autoAddCredentials
     };
 }
+
+export function getWindowOrThrow(): any {
+    if (typeof window === "undefined") {
```

Comment:
Any reason to use `typeof window === "undefined"` as opposed to `window === undefined`?

Reply:
Yes, if we do `window === undefined`, trying to access window object throws the `Reference Error` which we are trying to avoid.